catroid/src/androidTest/java/org/catrobat/catroid/test/content/sprite/SpriteTest.java

```diff
@@ -114,4 +120,16 @@ public void testUserVariableVisibilityOfLocalVariablesInDifferentScenes() {
 		userVariable = sprite2.getUserVariable(variableName);
 		assertTrue(userVariable.getVisible());
 	}
+
+	@Test
+	public void spriteEqualBackgroundTest() {
+		sprite.setName("Hintergrund");
+		updateLocale(ApplicationProvider.getApplicationContext(), new Locale("de"));
```

Comment:
Do you need the Locale 'de' here? Or could you also just use the default one and rewrite the test.